client/end.py

```diff
@@ -0,0 +1,29 @@
+from definitions import *
+import pygame as pg
+
+from buttonPress import Button
+
+def endScreen(screen, address):
+    print("hello world")
+    button_rejoin = Button(90, 30, "Rejoin", (50,130))
+    backToLogin = Button(200, 30, "Return to Login Screen", (50,180))
+    
+    done = False
+    while not done:
+        for event in pg.event.get():
+            if event.type == pg.QUIT:
+                sys.exit(0)
+            elif event.type == pg.MOUSEBUTTONDOWN:
+                button_rejoin.active = True
```

Comment:
Simply making the active status True because the mouse is clicked anywhere on the screen is incorrect. The button should become active once it is clicked on, and never any other time.